apps/lulesh/lulesh-simit.cc

```diff
@@ -0,0 +1,535 @@
+/*
```

Comment:
This file creates several clang warnings because braces are used to set a scalar element:
```
/Users/fred/Dropbox/projects/simit/simit/apps/lulesh/lulesh-simit.cc:296:29: warning: braces around scalar initializer
      [-Wbraced-scalar-init]
             elemMass.set(elem,{locDom->elemMass(nidx)});
```